examples/index.html

```diff
@@ -224,7 +221,18 @@ <h2>Improve this tool</h2>
       var sequencer;
     })
   </script>
-
+  <script src="uppy.js"></script>
+    <script>
+      var uppy = Uppy.Core()
+        .use(Uppy.Dashboard, {
+          inline: true,
+          target: '#dropzone'
+        })
+        .use(Uppy.Tus, {endpoint: 'https://master.tus.io/files/'})
```

Comment:
Oh... is this uploading them? Actually we don't need to upload, we can just use them locally, is that an option?

Reply:
Is there a way to pass the image to a function instead?

Reply:
Yup i forgot to remove them. :smile: